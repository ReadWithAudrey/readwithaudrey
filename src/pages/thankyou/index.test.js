import React from 'react'
import renderer from 'react-test-renderer'
import Thankyou from './index.js'

describe('thankyou page snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<Thankyou />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
