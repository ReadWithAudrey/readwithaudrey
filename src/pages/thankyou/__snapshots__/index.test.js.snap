// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`thankyou page snapshot test renders correctly 1`] = `
Array [
  <header
    className="sc-bdVaJa flex items-center justify-between ph3 montserrat white bg-green w-100 f3 chiVga"
  >
    <div
      className="flex items-center"
      id="header-left"
    >
      <img
        alt="book icon"
        className="h2 ph2"
        src="test-file-stub"
      />
      <Link
        to="/"
      >
        Audrey
      </Link>
    </div>
    <Link
      className="w-third"
      to="/Form1/"
    >
      <button
        className="sc-bZQynM bg-green hover-bg-pink pa2 mv2 white br2 w-100 f5 montserrat krNSkU"
      >
        Join Us
      </button>
    </Link>
  </header>,
  <div
    className="flex flex-column items-center justify-center page-padding ph4"
  >
    <h1
      className="f3 pink tc montserrat mb3"
    >
      Thank You
    </h1>
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      Thank you! We’ve received your registration and we will begin to pair you up a reading partner we think you'll connect well with shortly.
    </p>
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      We'll email you to introduce you and your reading partner once this is complete.
    </p>
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      If you have any questions in the meantime, please do not hesitate to contact us at
    </p>
    <a
      href="mailto:readwithaudrey@gmail.com"
    >
      readwithaudrey@gmail.com
    </a>
    <h2
      className="f3 pink tc montserrat mb3"
    >
      Help Spread The Word
    </h2>
    <p
      className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
    >
      Why not share the story of Audrey and help us spread the word?
    </p>
    <h3
      className="f5 pink tc montserrat mb3"
    >
      Share On
    </h3>
    <img
      alt="Facebook"
      className="fl"
      src="test-file-stub"
    />
    <img
      alt="Twitter"
      className="fl"
      src="test-file-stub"
    />
    <img
      alt="Email"
      className="fl"
      src="test-file-stub"
    />
    <Link
      to="/"
    >
      <p
        className="sc-jTzLTM montserrat f5 tc lh-copy dDfTnQ"
      >
        Go back to the homepage
      </p>
    </Link>
  </div>,
  <footer
    className="sc-bwzfXH flex flex-column justify-center items-center montserrat bg-green w-100 bt bw2 b--green uTxCW"
  >
    <span
      className="sc-htpNat flex justify-around items-center bg-white black w-100  pa2 jbutSu"
    >
      © Audrey
      <Link
        className="no-underline black ph1"
        to="/terms/"
      >
        Terms
      </Link>
      <Link
        className="no-underline black ph1"
        to="/privacy/"
      >
        Privacy
      </Link>
      <Link
        className="no-underline black ph1"
        to="/contact/"
      >
        Contact
      </Link>
    </span>
  </footer>,
]
`;
